Symfony/CS/Tests/Fixer/PSR2/ClassDefinitionFixerTest.php

```diff
@@ -154,6 +154,14 @@ public function provideAnonymousClassesCases()
                 "<?php \$a = new class(10) extends SomeClass implements SomeInterface, D\n{};",
                 "<?php \$a = new    class(10)     extends\nSomeClass\timplements    SomeInterface, D {};",
             ),
+            array(
+                "<?php \$a = new class(\$this->prop)\n{};",
+                '<?php $a = new class(   $this->prop   ){};',
```

Comment:
Stopping the fixer to do that should be relative issue, 
changing it to format to PSR 12 (including multiple line support with `use` case) will take some work.
Hope to look into this soon.